_aggregations/pipeline/avg-bucket.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Average bucket
+parent: Pipeline aggregations
+nav_order: 10
+redirect_from:
+  - /query-dsl/aggregations/pipeline-agg#avg_bucket-sum_bucket-min_bucket-max_bucket/
+  - /query-dsl/aggregations/pipeline/avg-bucket/
+---
+
+# Average bucket aggregations
+
+The `avg_bucket` aggregation is a sibling aggregation that calculates the average of a metric in each bucket of a previous aggregation.
+
+The specified metric must be numeric and the sibling aggregation must be a multi-bucket aggregation.
+
+## Parameters
+
+The `avg_bucket` aggregation takes the following parameters.
+
+| Parameter             | Required/Optional | Data type       | Description |
+| :--                   | :--               |  :--            | :--         |
+| `buckets_path`        | Required          | String          | The path of the aggregation buckets to be aggregated. See [Pipeline aggregations]({{site.url}}{{site.baseurl}}/aggregations/pipeline/index#pipeline-aggregation-syntax). |
+| `gap_policy`          | Optional          | String          | The policy to apply to missing data. Valid values are `skip`, `insert_zeros`, and `keep_values`. Default is `skip`. |
+| `format`              | Optional          | String          | A [DecimalFormat](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/text/DecimalFormat.html) formatting string. Returns the formatted output in the aggregation's `value_as_string` property. |
+
+## Example
+
+The following example creates a date histogram with a one-month interval from the OpenSearch Dashboards e-commerce sample data. The `sum` sub-aggregation calculates the sum of all bytes for each month. Finally, the `avg_bucket` aggregation calculates the average number of bytes per month from these sums:
```

Comment:
`average bytes per month` reads better instead of `average number of bytes per month`

`sum of bytes` reads better instead of `sum of all bytes`